docs/source/index.md

```diff
@@ -27,6 +31,7 @@ Apollo Android runs on the following platforms:
 * Android API level 15+
 * JDK 8+
 * iOS 13+
+* JavaScript
```

Comment:
I changed this in `dev-3.x` already: https://github.com/apollographql/apollo-android/tree/dev-3.x#requirements